package.json

```diff
@@ -91,31 +93,33 @@
     "pusher-js": "^5.0.3",
     "raf": "^3.4.1",
     "rails-erb-loader": "^5.0.2",
-    "react": "^16.12.0",
-    "react-clipboard.js": "^2.0.16",
-    "react-color": "^2.17.3",
-    "react-datepicker": "^2.10.1",
-    "react-dom": "^16.12.0",
-    "react-dropzone": "^10.2.1",
+    "react": "^16.8.0",
+    "react-beautiful-dnd": "^12.0.0",
+    "react-clipboard.js": "^2.0.1",
+    "react-color": "^2.11",
+    "react-datepicker": "^0.46.0",
     "react-mentions": "^3.1.1",
-    "react-redux": "^7.1.3",
-    "react-tag-autocomplete": "^5.11.2",
-    "redux": "^4.0.4",
-    "redux-devtools-extension": "^2.13.8",
-    "redux-thunk": "^2.3.0",
+    "react-dom": "^16.8.0",
```

Comment:
Any reason for regress the version of these dependencies?

Reply:
No reason, I'll update it.